### PR TITLE
[IMP] theme_buzzy: revamp the theme

### PR DESCRIPTION
**Pull request by Brieuc-brd**
task-4178061
Part of task-4177975

| Before | After |
|--------|--------|
| image | buzzy_screenshot |

### CODE REVIEW

**Comment by Brieuc-brd** [comment]
Conflict resolved

File: theme_buzzy/views/snippets/s_image_punchy.xml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+        <attribute name="data-shape">web_editor/geometric/geo_slanted</attribute>
+        <attribute name="data-original-mimetype">image/jpeg</attribute>
+        <attribute name="data-file-name">s_image_punchy_default_image.svg</attribute>
+        <attribute name="data-shape-colors">;;;;</attribute>
+    </xpath>
+    <xpath expr="//h2" position="replace" mode="inner">
+        DRIVING INNOVATION
+    </xpath>
+    <xpath expr="(//div[hasclass('o_grid_item')])[2]" position="attributes">
+        <attribute name="class" add="g-height-5" remove="g-height-6"/>
```
Comment (bso-odoo) [inline theme_buzzy/views/snippets/s_image_punchy.xml:16]: Separator might be missing: a comma appears in the DOM before `g-height-5` and `g-height-6` is not removed.

File: theme_buzzy/views/snippets/s_cta_box.xml [hunk 2]
```diff
@@ -4,8 +4,17 @@
 <template id="s_cta_box" inherit_id="website.s_cta_box">
     <!-- Card -->
     <xpath expr="//div[hasclass('card')]" position="attributes">
-        <attribute name="class" add="o_cc5" remove="o_cc4" separator=" "/>
+        <attribute name="class" add="o_cc3" remove="o_cc4" separator=" "/>
+        <attribute name="style" add="border-radius: 12px !important" separator=" "/>
```
Comment (bso-odoo) [inline theme_buzzy/views/snippets/s_cta_box.xml:8]: Shouldn't the `style` separator be `;` ? (several times in this PR)

**Review by bso-odoo — COMMENTED** [review]
Also, there is a contrast issue on `s_image_title` (and it has no image)
image

**Comment by Brieuc-brd** [comment]
Hey @bso-odoo 👋 Thanks for your review 👍 
I updated the branch, what do you think ?

The diff : https://github.com/odoo/design-themes/compare/cb47cf5b062566c880c2bb54508e6f792f784d16..6764c05fc4d99f147c23797136d27bf636598028

**Comment by bso-odoo** [comment]
LGTM

I found these slightly poor shape connections when adding a page... (2 of them, the first one is fine)
image

Tell me if you want to adapt something about them, otherwise I'll r+ the current version.

**Comment by Brieuc-brd** [comment]
Good catch @bso-odoo, I've made some adaptations ✅

**Comment by bso-odoo** [comment]
The contrast of the last block on configurator-generated home page is not great:
image

It is the only issue I found.

**Comment by Brieuc-brd** [comment]
Ok I see, the issue is rather related to the palette used but I agree that it doesn't work when using the web configurator (we expect a good result without having to edit).
I adapted the shape for the configurator and kept the initial shape for the theme 👍 

The diff : https://github.com/odoo/design-themes/compare/65126f3a2e58b7fff8dec1aaab883f24a9b882a5..079a18aedc63f21b51958c1258b1288c6ec9abd5

**Comment by bso-odoo** [comment]
@robodoo r+

**Pull request closed without merging**